python/pyarrow/table.pxi

```diff
@@ -5383,11 +5383,9 @@ list[tuple(str, str, FunctionOptions)]
             for col_name, (aggr_name, _) in zip(columns, group_by_aggrs)
         ] + self.keys
 
-        result = _pc()._group_by(
+        agg_tables = []
```

Comment:
Left-over variable of the previous iteration?

Reply:
I've removed this.